### ISSUE TITLE
supported VM OS versions?

### ISSUE THREAD

**Issue by JiaoSun123** [Azure/azure-storage-fuse#783]
### Which OS (please include version) are you using?
Linux (ubuntu 22.04)

### What problem was encountered?
All packages are up to date.
root@ubuntu7:/home/azureuser# sudo apt-get install blobfuse fuse3
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package blobfuse

### Have you found a mitigation/solution?
No

### If relevant, please share your mount command.
I searched on the internet about this error regarding my Ubuntu version. I think the package is not available for it. It would be great if we can get a list of supported OS versions before we start.

**Comment by gapra-msft** [comment]
Hi @JiaoSun123, 
You can find the supported OS versions here for any given release https://github.com/Azure/azure-storage-fuse/releases/tag/blobfuse-1.4.3

**Comment by gapra-msft** [comment]
I will close this issue now as I believe your question has been answered. If you have further questions, feel free to reopen this or create a new issue and we are happy to help!
